ros/src/computing/perception/detection/packages/lidar_tracker/nodes/obj_fusion/obj_fusion.cpp

```diff
@@ -56,54 +58,80 @@ static double euclid_distance(const geometry_msgs::Point pos1,
 } /* static double distance() */
 
 /* fusion reprojected position and pointcloud centroids */
-static void fusion_objects(void) {
-  obj_label_t obj_label_current;
-  std::vector<autoware_msgs::CloudCluster> v_cloud_cluster_current;
+void fusion_cb(const autoware_msgs::obj_label::ConstPtr &obj_label_msg,
+  const autoware_msgs::CloudClusterArray::ConstPtr &in_cloud_cluster_array_ptr) {
```

Comment:
I think that Autoware follows ROS cpp style guide. In there braces go into a new line: http://wiki.ros.org/CppStyleGuide#Formatting.

@yk-fujii is that correct?